anaflow/flow/Neuman.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+"""
+Anaflow subpackage providing the Neuman equation for homogeneous aquifer.
+
+.. currentmodule:: anaflow.flow.Neuman
+
+The following functions are provided
+
+.. autosummary::
+
+"""
+# pylint: disable=C0103
+import numpy as np
+from scipy.special import k0, k1
+from scipy.integrate import quad as integ
+
+import matplotlib.pyplot as plt
+
+from anaflow.tools.laplace import get_lap_inv
+from anaflow.flow.laplace import grf_laplace
+from anaflow.tools.special import Shaper, sph_surf
+
+__all__ = []
+
+
+def neuman(
+        s,
+        rad,
+        storage,
+        transmissivity,
+        sat_thickness=52.0,
+        rate=-1e-4,
+        d=0.6,
+        l=12.6,
+        rw=0.1,
+        kz=0.14,
+        kr=0.23,
+        ss=4.27e-5,
+        n_numbers=52,
+):
+    """
+        The Neuman solution.
+
+
+        Parameters
+        ----------
+    s : :class:`numpy.ndarray`
+        Array with all time-points where the function should be evaluated in the Laplace space.
+    rad : :class:`numpy.ndarray`
+        Array with all radii where the function should be evaluated.
+    storage : :class:`float`
+        Storage of the aquifer.
+    transmissivity : :class:`float`
+        Geometric-mean transmissivity.
+    sat_thickness : :class:`float`, optional
+        Saturated thickness of the aquifer.
+    rate : :class:`float`, optional
+        Pumpingrate at the well. Default: -1e-4
+    d : :class:`float`, optional
+        Vertical distance from initial water table to top of
+        pumped well screen
```

Comment:
Would use `screen_size` instead and calculate `d = well_depth - screen_size` to be in line with https://github.com/GeoStat-Framework/welltestpy/issues/18